go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -67,11 +70,58 @@ func (n *NodeController) addOrUpdatePod(pod *kapi.Pod) error {
 	if err != nil {
 		return fmt.Errorf("failed to add flows for pod %s/%s: %v", pod.Namespace, pod.Name, err)
 	}
+	subnet, err := getLocalNodeSubnet(n.nodeName)
+	if err != nil {
+		return err
+	}
+	namespace, err := n.kube.KClient.CoreV1().Namespaces().Get(pod.Namespace, metav1.GetOptions{})
+	if err != nil {
+		klog.V(5).Infof("unable to get namespace %s annotations", pod.Namespace)
+	}
+	if namespace == nil {
```

Comment:
@rcarrillocruz shouldn't this if just be combined with the above `if err != nil {`? I don't see the point of having both.  Also, I can't ever think of a situation where namespace == nil && err == nil...